styles/_utils.scss

```diff
@@ -2,6 +2,7 @@
 @import 'mixins/card';
 @import 'mixins/spacing';
 @import 'mixins/border';
+@import './theme.base';
```

Comment:
why we need this import here inside utils ?

Reply:
Moved the them.base code to the theme.scss file.